lab3/src/main.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+from data_humans import DataHumans
+
+
+"""
+main.py --destination_folder='D:\WinUsers\Lera\Documents\summerlabs\pythonrep\lab3\strang2 ' --file_name=fyufyff2 --gender_filt=male
+"""
+
+
+# использовать argparse
+parser = argparse.ArgumentParser()
+parser.add_argument("--destination_folder", type=str, required=True)
+parser.add_argument("--file_name", type=str, required=True)
+parser.add_argument("-g", "--gender_filt", type=str)
+parser.add_argument("-n", "--numb_of_rows_filt", type=int)
+parser.add_argument("--log_level", type=str)
+args = parser.parse_args()
+
+import pdb
+
+pdb.set_trace()
+
+output_file = DataHumans(args.destination_folder, args.file_name)
+if args.numb_of_rows_filt is True:
```

Comment:
you don't need to compare var wtih True.
do this:
```suggestion
if args.numb_of_rows_filt:
```